nodejs/week1/radiusWarmup/radiusWarmup.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="warmup.js"></script>
+</body>
+</html>
```

Comment:
great,.,i think it escaped me because i was first running node from GIT bash instead of my terminal,.,but now I will use the terminal only to avoid such errors